src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -35,30 +37,31 @@
             + PREFIX_TIME + "1240 "
             + PREFIX_DESCRIPTION + "We went to the beach!";
 
+    public static final String MESSAGE_CONSTRAINTS =
+            "Meetings should have already happened before they are recorded. Meeting description should not be empty";
```

Comment:
Can reword to "Meetings are records of past events, and should not be in the future. Meeting description should not be empty."